content/wissen/test4/index.md

```diff
@@ -0,0 +1,32 @@
+---
+draft: true
+
+title: "Test page 4"
+description: "How to set up a new branch, make changes and make a pullrequest"
+topics: ["test", "abc"]
+authors:
+  - "fackerl"
+series: ["Strommärkte"]
+series_order: 2
+weight: 2
```

Comment:
This probably conflicts with the already existing [test1 article](https://github.com/ait-energy/en.ergie.at/blob/21e58b9678dbe6b4f9110149dc8041c4a9602fa3/content/wissen/test1/index.md), since that is "2nd article in Strommärkte" already.

Either remove that (`#` works as comment in the front-matter) or update the order & weight (e.g., to 3).